pkg/annotations/ingress_annotations.go

```diff
@@ -59,9 +59,8 @@ const (
 	// that this is a gateway meant for the application gateway ingress controller.
 	IstioGatewayKey = "appgw.ingress.istio.io/v1alpha3"
 
-	// ApplicationGatewayIngressClass defines the value of the `IngressClassKey` and `IstioGatewayKey`
-	// annotations that will tell the ingress controller whether it should act on this ingress resource or not.
-	ApplicationGatewayIngressClass = "azure/application-gateway"
+	//DefApplicationGatewayIngressClass defines the default app gateway ingress value
+	DefApplicationGatewayIngressClass = "azure/application-gateway"
```

Comment:
Why don't we just name it "DefaultApplicationGatewayIngressClass"?

Reply:
Name getting pretty long. Will change it.

Reply:
this can also be `DefaultIngressClass`

Reply:
Even better!